components/clamav/files/var/svc/manifest/site/freshclam.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0'?>
```

Comment:
If I can ensure that a next iteration will be scheduled (despite any intermittent errors), and two iterations won't race in conflict at the same time - why not. As a first shot that gets the job done, a loop which sequentially updates the database and can be stopped/started as an smf service seemed sufficient until proven otherwise. (Might need corrections about service type though, e.g. transient maybe...)